openslides/agenda/static/js/agenda/agenda.js

```diff
@@ -40,30 +62,105 @@ angular.module('OpenSlidesApp.agenda', [])
     });
 })
 
-.controller('ItemListCtrl', function($scope, Agenda, i18n) {
+.controller('ItemListCtrl', function($scope, $http, Agenda) {
     Agenda.bindAll({}, $scope, 'items');
-    $scope.test_plural = i18n.ngettext('test', 'tests', 2);
-    $scope.test_singular = i18n.ngettext('test', 'tests', 1);
+
+    // get full agenda tree
+    $http.get('/rest/agenda/item/tree')
```

Comment:
Please put this line inside the resolve-object in line 17 (I added also a comment to your commit with a description how to do this)